newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -62,10 +71,33 @@ export default function SummaryPage() {
     _sendResultEmails(newdle.code);
   }, [setMailModalOpen, newdle, _sendResultEmails]);
 
+  const handleDeletionModalClose = useCallback(() => {
+    setDeletionModalOpen(false);
+  }, [setDeletionModalOpen]);
+
+  const handleDeletionModalConfirm = useCallback(async () => {
+    if (sendDeletionMail) {
+      _sendDeletionEmails(newdle.code, comment);
```

Comment:
```suggestion
      _sendDeletionEmails(newdle.code, comment.trim());
```

just in case people enter just whitespace